test/common.js

```diff
@@ -12,6 +12,7 @@
 /* globals assert */
 
 process.env.NODE_ENV = 'test';
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
```

Comment:
The WebSocket test needs to connect to the original https server because `express-ws` is bound to it. All the other tests are okay because `chai-http` creates a new http server by default.